pkg/ranger/content.go

```diff
@@ -209,6 +209,10 @@ func isZeroTime(t time.Time) bool {
 }
 
 func setLastModified(w http.ResponseWriter, modtime time.Time) {
+	if w == nil {
```

Comment:
what's the added benefit here of returning nothing? Just curious on what happens if that happens? Should we log something? or?

Reply:
I added this check after corner case when http.ResponseWriter is nil.
Without nil check, go fails with SIGSEGW.

I wonder is it possible to http.ResponseWriter to be nil in prod?